src/components/footer/index.js

```diff
@@ -25,20 +29,26 @@ function Footer() {
     return (
         <Card className="wpui-footer" size="large" isBorderless>
             <CardBody>
-                <VStack className="sec-1" alignment="center" spacing={8}>
-                    <Heading color='white'>WPUI Components</Heading>
+                <VStack className="sec-1" alignment="center" spacing={4}>
```

Comment:
```suggestion
                <VStack className="footer-content" alignment="center" spacing={4}>
```

class can be more descriptive!